dts/bindings/vendor-prefixes.txt

```diff
@@ -647,7 +647,7 @@ vxt	VXT Ltd
 wand	Wandbord (Technexion)
 waveshare	Waveshare Electronics
 wd	Western Digital Corp.
-we	Würth Elektronik GmbH.
+we	Würth Elektronik GmbH
```

Comment:
This change should be in its own commit.

Reply:
Ok, I've removed this change from the commit.